scripts/var_props.py

```diff
@@ -128,6 +128,44 @@ def find_vertical_dimension(dims):
     # end for
     return (var_vdim, vindex)
 
+########################################################################
+def local_name_to_diag_name(prop_dict, context=None):
+########################################################################
+    """
+    Translate a local_name to its default diagnostic name.
+    Currently, this is just equal to the local name. If no local name
+    exists in the property dictionary, a truncation of the standard
+    name is used. (256 characters = max length of NetCDF variable name)
+    >>> local_name_to_diag_name({'local_name':'foo', 'standard_nam':'cloud_optical_depth'})
+    'foo'
+    >>> local_name_to_diag_name({'standard_name':'cloud_optical_depth_layers_from_0p55mu_to_0p99mu'})
+    'cloud_optical_depth_layers_from_0p55mu_to_0p99mu'
+    >>> local_name_to_diag_name({'units':'km'}) #doctest: +ELLIPSIS
+    Traceback (most recent call last):
+    ...
+    parse_source.CCPPError: No standard name or local name to convert to diagnostic name
+    """
+    diag_name = None
+    if 'local_name' in prop_dict:
+        locname = prop_dict['local_name']
+        if locname:
+            diag_name = prop_dict['local_name']
+        # end if
+    elif 'standard_name' in prop_dict:
+        stdname = prop_dict['standard_name']
+        if stdname:
+            maxlen = 256
+            diag_name = stdname[:maxlen] if len(stdname) > maxlen else stdname
```

Comment:
Is this necessary?
From https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str
"The slice of s from i to j is defined as the sequence of items with index k such that i <= k < j. If i or j is greater than len(s), use len(s)."
```suggestion
            maxlen = 256
            diag_name = stdname[:maxlen]
```

Reply:
@peverwhee Do you have an answer to this question?

Reply:
yes sorry for the delay! I've pushed up @gold2718 's suggested change